internal/rest/get_league_of_legends_stats.go

```diff
@@ -1,19 +1,19 @@
 package rest
 
 import (
+	"github.com/dct-tournaments/gamer-stats-api/pkg/leagueoflegends"
+	"github.com/gin-gonic/gin"
 	"net/http"
 	"strconv"
 	"time"
-
-	"github.com/dct-tournaments/gamer-stats-api/pkg/leagueoflegends"
-	"github.com/gin-gonic/gin"
 )
 
 type QueueType string
 
 const (
 	RankedQueueType QueueType = "ranked"
 	AllQueueType    QueueType = "all"
+	Aram            QueueType = "aram"
```

Comment:
Rename to follow the same pattern

```suggestion
	AramQueueType            QueueType = "aram"
```